src/routes/(auth)/signin/+page.svelte

```diff
@@ -1,4 +1,92 @@
+<script lang="ts">
+    import Input from "$lib/components/composites/input/Input.svelte";
+    import PasswordInput from "$lib/components/composites/input/PasswordInput.svelte";
+    import { Button } from "$lib/components/primitives/button/index.js";
+    import * as Card from "$lib/components/primitives/card/index.js";
+    import { backendService } from "$lib/grpc-api";
+    import { Schema } from "$lib/schemas";
+    import type { ApiError } from "$lib/model/general";
+    import ErrorAlert from "$lib/components/composites/utils/ErrorAlert.svelte";
+    import { StatusCodes } from "$lib/model/error-codes";
+    import { goto } from "$app/navigation";
+
+    let emailInput: Input;
+    let passwordInput: PasswordInput;
+
+    let registrationError: ApiError | undefined = $state(undefined);
+
+    async function handleSubmit(event: Event) {
+        event.preventDefault();
+
+        const isEmailValid = emailInput.validate();
+        if (!isEmailValid) {
+            return;
+        }
+
+        const userData = {
+            email: emailInput.getValue(),
+            password: passwordInput.getValue(),
+        };
+
+        backendService
+            .login(userData)
+            .then(() => {
+                console.log("User successfully logged in.");
+                goto("/");
```

Comment:
Same as in the signout. Maybe remove the signin page from the history?

```suggestion
                goto("/", { replaceState: true });
```